cypress/integration/operator_spec.js

```diff
@@ -31,7 +36,68 @@ describe('Operator creation basic tests', function () {
         cy.contains('Omat palvelutiedot');
     });
 
-    it('should validate and invalidate business id', function () {
+    it('should navigate to add new operator', function () {
+        cy.contains('Omat palvelutiedot');
+        cy.get('#btn-add-new-transport-operator').click();
+        cy.contains('Lisää palveluntuottaja');
+    });
+
+    it('should not be able to save operator with invalid business id', function () {
+        cy.contains('Omat palvelutiedot');
+        cy.get('#btn-add-new-transport-operator').click();
+        cy.get('#input-business-id').type('1234');
+        // Check warning
+        cy.contains(testOp1.businessIdInvalidWarding);
+
+        // Save button should be disabled
+        cy.get('#btn-operator-save').should('be.disabled');
+        cy.visit('/#/');
+        cy.get('#btn-confirm-leave-page').click();
+    });
+
+    it('should add new operator', function () {
+
+        cy.contains('Omat palvelutiedot');
+
+        // Start adding new operator
+        cy.get('#btn-add-new-transport-operator').click();
+
+        // Business id is already added
+        cy.get('#input-business-id').type('1234567-8');
+        // Check warning
+        cy.contains(testOp1.businessIdExistsWarning);
+
+        // Try creating random business-id
+        cy.get('#input-business-id').clear(); // Clear business-id
+        cy.get('#input-business-id').type(randomBusinessid());
```

Comment:
While which approach to take is a matter of debate, using a random id may lead to test sometimes failing and if it's rerun, it passes. E.g. if random id outputs '1234567-8'. 
Due to human laziness the issue might be overlooked then.
I'd prefer the apprach that a running test should have always the same result, so the valid business id should be hard-coded.
That way it's also possible to cover systematically the different types of id validation, e.g. length/characters/exists/other.